doc/source/changes/version_0_24.rst.inc

```diff
@@ -0,0 +1,20 @@
+New features
+------------
+
+* added a feature (see the :ref:`miscellaneous section <misc>` for details).
+
+* added another feature.
+
+.. _misc:
+
+Miscellaneous improvements
+--------------------------
+
+* improved something.
+
+Fixes
+-----
+
+* fixed title argument of `ndtest` creation function: title was not passed to the returned array.
+
+* fixed documentation of Session: all methods are now documented (closes :issue:`277`)
```

Comment:
I consider this a misc improvement, not a fix. Not documenting was not done by mistake.